test/quantization/test_quantize_script.py

```diff
@@ -1412,7 +1419,20 @@ def forward(self, x, y):
 
         FileCheck().check_not("aten::cat") \
                    .check("quantized::cat") \
-                   .run(m.graph_for(data, data))
+                   .run(m.graph)
+
+        # non quantized cat
+        m = torch.jit.script(NonQuantizedCat()).eval()
```

Comment:
In this case, what is the expected behavior? Would you quantize all the inputs to non-quantized cat  and quantize the output too? In general, dont you quantize the input to a model?

Reply:
not we don't quantize the inputs to cat in this case, the output of cat is only quantized when all the inputs are quantized.

Reply:
we will not quantize the input of cat in this case.